src/group-configurations/messages.js

```diff
@@ -0,0 +1,41 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  headingTitle: {
+    id: 'course-authoring.group-configurations.heading-title',
+    defaultMessage: 'Group configurations',
+    description: 'Title for the heading of the group configurations section.',
+  },
+  headingSubtitle: {
+    id: 'course-authoring.group-configurations.heading-sub-title',
+    defaultMessage: 'Settings',
+    description: 'Subtitle for the heading of the group configurations section.',
+  },
+  containsGroups: {
+    id: 'course-authoring.group-configurations.container.contains-groups',
+    defaultMessage: 'Contains {len} groups',
+    description: 'Message indicating the number of groups contained within a container.',
+  },
+  containsGroup: {
+    id: 'course-authoring.group-configurations.container.contains-group',
+    defaultMessage: 'Contains 1 group',
+    description: 'Message indicating that there is only one group contained within a container.',
+  },
```

Comment:
It seems we have only two options. There are groups with `contains groups` and `contains group` without a counter. Am I understanding this correctly?

Reply:
This update utilizes the plural function available with `intl()` based on the value of `len`. If `len` equals one, it will use "group". If 'len' equals any other number, it will use "groups". This is a reduces the amount of messages required.

Reply:
```
containsGroups: {
    id: 'course-authoring.group-configurations.container.contains-groups',
    defaultMessage: 'Contains {len, plural, one {group} other {groups}}',
    description: 'Message indicating the number of groups contained within a container.',
  },
```
The `defaultMessage` should be 
```
containsGroups: {
    id: 'course-authoring.group-configurations.container.contains-groups',
    defaultMessage: 'Contains {len, plural, one {1 group} other {{len} groups}}',
    description: 'Message indicating the number of groups contained within a container.',
  },
```